docs/fsharp/style-guide/component-design-guidelines.md

```diff
@@ -68,7 +68,7 @@ The following table follows .NET naming and capitalization conventions. There ar
 |-----------|------|------|----------|-------|
 | Concrete types | PascalCase | Noun/ adjective | List, Double, Complex | Concrete types are structs, classes, enumerations, delegates, records, and unions. Though type names are traditionally lowercase in OCaml, F# has adopted the .NET naming scheme for types.
 | DLLs           | PascalCase |                 | Fabrikam.Core.dll |  |
-| Union tags     | PascalCase | Noun | Some, Add, Success | Do not use a prefix in public APIs. Optionally use a prefix when internal, such as ```type Teams = TAlpha | TBeta | TDelta.``` |
+| Union tags     | PascalCase | Noun | Some, Add, Success | Do not use a prefix in public APIs. Optionally use a prefix when internal, such as ```type Teams = TAlpha \| TBeta \| TDelta.``` |
```

Comment:
This doesn't fix the problem, @v-makoud. The vertical bars render as `\|`. Removing the extra two backticks may resolve the problem. I'll do a new commit with that change. @mairaw and @BillWagner, do you know whether that should address the issue. Here is an example of the original localized content:

![image](https://user-images.githubusercontent.com/10886961/49599602-49697b80-f936-11e8-8cef-332f5dbf9298.png)